coursera_autograder/commands/update_resource_limits.py

```diff
@@ -137,7 +137,6 @@ def setup_registration_parser(parser):
     parser.add_argument(
         '--grader-cpu',
         default=None,
-        choices=['1', '2', '4'],
```

Comment:
Will we add in other validation to make sure the choice is valid?

Reply:
https://coursera.atlassian.net/browse/HOL-3619 I don't think so. The validation will only be in our backend.